boards/weact/stm32g474_core/board.cmake

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(dfu-util "--pid=0483:df11" "--alt=0" "--dfuse")
+board_runner_args(stm32cubeprogrammer "--port=swd" "--reset-mode=hw")
+include(${ZEPHYR_BASE}/boards/common/stm32cubeprogrammer.board.cmake)
```

Comment:
First line is the default runner, and from what I understand only working solution among those is dfu.
A clean up is required in this file